src/cloud-functions.ts

```diff
@@ -609,15 +622,14 @@ export function optionsToEndpoint(
     'serviceAccount',
     (sa) => sa
   );
-  if (options.vpcConnector) {
-    const vpc: ManifestEndpoint['vpc'] = { connector: options.vpcConnector };
+  if (options?.vpcConnector) {
+    endpoint.vpc = { connector: options.vpcConnector };
     convertIfPresent(
-      vpc,
+      endpoint.vpc,
       options,
       'egressSettings',
       'vpcConnectorEgressSettings'
     );
-    endpoint.vpc = vpc;
   }
   convertIfPresent(endpoint, options, 'availableMemoryMb', 'memory', (mem) => {
     const memoryLookup = {
```

Comment:
FML that Entity has already been defined to have memoryMB yet Run now uses these friendly names (almost) directly.